src/coreclr/vm/amd64/AsmHelpers.asm

```diff
@@ -1019,6 +1021,8 @@ JumpToMoreStack:
     add rsp, 28h
     ret
     jmp r11
+ALTERNATE_ENTRY JIT_GreenThreadMoreStack_end ; When yielding, we will jmp back to this location with the registers back in their original config
```

Comment:
This comment is a copy/paste mistake.  (or I'm very confused about how this works)

Reply:
Very much a copy past mistake.